doc/tutorials/intermediate/structure_data_store.md

```diff
@@ -0,0 +1,323 @@
+# Structure with a DataStore
+
+In this tutorial you will learn how to use the `DataStore` *design pattern*:
+
+- The `DataStore` design pattern has proven quite successful across a wide range of use cases.
+- The `DataStore` component receives `data` and `filters`. Based on the `data` and `filters` it *transforms* the data.
+- One or more `View` components consumes the *transformed* data.
+- The components can be (re-)used for data exploration in notebooks and for building apps.
+
+:::{admonition} Note
+When building a larger application we generally recommend using a *class based construction* and following best practices for object-oriented programming, specifically composition.
+:::
+
+```{pyodide}
+import panel as pn
+pn.extension('tabulator', 'vega', throttled=True)
+```
+
+<div class="header-box" style="box-shadow: rgba(50, 50, 93, 0.25) 0px 6px 12px -2px, rgba(0, 0, 0, 0.3) 0px 3px 7px -3px; padding: 5px 10px; border-left: 4px solid green;">
+
+## Install the dependencies
+
+Please make sure [Altair](https://altair-viz.github.io/), [Pandas](https://pandas.pydata.org) and [PyArrow](https://arrow.apache.org/docs/python/index.html) are installed.
+
+::::{tab-set}
+
+:::{tab-item} conda
+:sync: conda
+
+``` bash
+conda install -y -c conda-forge altair pandas pyarrow
+```
+:::
+
+:::{tab-item} pip
+:sync: pip
+
+``` bash
+pip install altair pandas pyarrow
```

Comment:
pandas should be included in panel I think?

Do we need to introduce altair? For tutorials, I think keeping them as lightweight as possible would be nice, without having to do extra work outside the notebook

Reply:
Altair is actually the most download python js-based plotting library. I would like to promote more that it works great with Panel. I do that here. I don't think that minor dependencies is a problem.